src/main/resources/caserefs-prod-cs-encrypted.csv

```diff
@@ -0,0 +1 @@
lYMFkdtOiQQAeawOc+yjyijP+LzimZ2IwmR13gXM1diHpMnJvsiOINHZ/w2fdva0CaUg9Lf3OePJFFT7BN+8EdR8li3Kz0Usk7gzX3M65tb2FDMOGQ6DGxAsqkrH7CNI/5szX4mDFw5p4UiJYrUpofQIp8sywYFtCUkotIBYgrrlyW1DLmyp0B73RAj26q1K0ozjqYYHe6KaVB8eIJig8WRvmFT58P7araftvS5oG+MdSEfC4LcI6O9u8UlO0k1PJCFbGJpbJ8PZExTAUAsgWCOzWjF/6cITL77JjzKX18mpX0xbKxg2pCNUmlOdRApFw8cy9IZu1XzLeja9S/pkBdsQYVUXMpItuMzbFL+nPfnf8g80gO+IA4eILegD1NysQC47FTy5oE5vCMPiXgUsGry1/iidHNZZjds3x1bve7TcGMRABUDv2aQ5ZgqUIlGQeZwGhBrU1DWZK6oTMg/7q7JTVKcOK0mdtFLbeqjCE6RZoDFcCi/9+EV7gUnhnDxP+dX8v0fF7zx5a2agvPbzNVOkcbZbdZa1lY3tgO7ibsqEuYj6IcF5gjKeSXJhQUrCiGFDO5FBi+I2BvHdFg0IOzlD2/sY7MRRtKgodCa4vX2A0T+xuFBm452iJsIqcAdDibfR04Qi7gW8YMoeWb0bxLIM6WxRFV3VlSA2cWO7ku4kGne1mM4JzpdBHpHuLZoboFRTmpUVZPIc1gCByqfYoHiy/sKot58htHj0w/pbiMj7LvLxxFIw2EpFe0hMeDpLjNHBYV9aWIpPa8TGHxGkkOA8mU4D4tBibVlYjfsZ0DuggjXDjbPaqxasIWeb0wDSPzi8ydPsqP0Hk8Nj9IbeJ8TmEr11MuUOqIbK3/VRktTDOVXMdwX+4wsCkDRVIkvNsXzX8HNKzXuyT8rdrZJQk+yqG3Zu61htiIhNIGVI1wtDw5YARpCVtdWc5U2EPvm/tfrr+Ikg4gfKfIz6ltBATHaYFzpHSWkYD8mpuMInvrblDthh3Va5onZI9ViL17zQXyfaD1a5EJdkhX3b62EqJcAY8i7012wsVsxZKbiUwpXl79mI2ILNynlChHubqFskFqY6Ejbg0JkY3KjNIDOSc5OzNEnkLxsN9ph22UaHNE9OkTmalVAwBB5c392ugAbXvgEFqDNRXUHhWqv8M2Xrtv3fjsH/XdTSEbBuQilRLK3TRXynlJAGBYoG4ZwRf9i+80rn3X5m26l8uXU9d5zQsIgmUEYp1gvzsR7UjwmIPwS5xsJtwgFMxL6duuAwQIbNxLZPul5edhxtRAl/g3uCQE+sLuhvq851tM8vjxcvO3wnKHdcUEiuR1CRMBdRwmvqNZwVQWiQKAPKDTbLIfUkCfwTukhKzqhV6+jaEK1f5oIMlRGmFEIkNO189i4C+y7SImejWmvAKT7oClt24+yTU5kG2kc2chvdZ8pN2DJlu3PdT9LQoc8t4MgQCfqpyC4+zG00BR4trIBjjDgWc4FP8jpN+vcGZ+N/tgUEWfcEE8Z4Se/z9SUoCF0UrCA8N/qKNjeWu0u5EXa6AuLWBUl+tJh4fqP+616S3LQS19cJoAaWUAHY0WPn+IOaYHsksbRyH8HQ+Snd/PqJc7+dpqdkb/68GkhydQL5Tzaa+GuffX0tIesivBWoSRlhUEnLa7qDAUT2d3JvIuhD9Nj+OUdLcUWLe2E2SDkD369Y869a8tOoFHL/OxvVnuUDfNkQa/q7CjPMyh6pL5Snz4fLc0oUjQ6dohFayoWPbbKGi8riVzou7kP38zO6qEDNYtiHjCG+/RPHvr1ix764INCxHoygdaLzcrskNemsLk7wDv2qW1IaU4OSJzT1ezFt72LjmH/RN4ah//MMvJ2rv6V75KWRFreVA+2X+8m4TgfsV/zWHxfNu2359WxzRbx7lNaN8afI76skqmlRI+YnrNmXfhSDMVRwvdcDopxvqu7c9unDCEn4QnGjtui+2smt5yUan5fp78dcAiWNvtYwdyeywSoTS0T/I0Wycquy+HxOYb32PweTe4bdz525MCl2cihR+PJfHZ8sUbAnn0z+2eCER8v7yfmrSq5lnNC5JEulFkMepWufdzT/a7ZjiAJHykl8bLd2BynXIu3D6ZE5qoN1jbx0efbGhyTLII/JjkS1guyeX0fLf3VbmrWdk6QJjM597/XR/zqwTR5zfqoea9dNDHl5+LU5y819dYtWxGynId29QmYC/3rT3zBYv6IK2a45ye8oXZ8hWG1VKkj130PNHWAAQLxeoN/I1+eRSvnm6GY/4AFdZyyPx+TURf/TxnIoF8sOHB7NyJGj8wanWJRfPzahVBk/VwzBdXykaDCbE/KsAAXpm23uAQ/Ncbnk6cN4oKZd98WjmU+jVzpzOLTIICPgOv4yHR5OH9U5FOFqUnhASMveczLbe7LJWUu4gZqzEGv5JpqdXB/MtK9U+rVuCIr5c10g21fQh67E3SrAKGrG+j45s57dqONQtLDbGpuW1SSXolOr0f5TJ1N59+SYQlq+j6ht0s3p+N8GF7yanZ5lZZ2Sne/2kQWI6xCEMi873ehipkn//ab+O/8jJCyF/M+3h4XVvwzNbwCTg5pPpRlm8zOp4B0FSq8OS/FEXeJy1c9vfS5k/0J8DICG+AogFfK7qFP5hcu6Vm8PBQ4c9uexJg9OCL6CHrGwOzbOn+iPAtkYDjQ+r39KvaNwVhxW2acRoCPXGzTvPG3fm8Hteyyo8er5ad6LnEhnQFmQxlI0SpfuqjQA4aTbM8sqltoWclL45cRXpS1JioQSa1w4rDrEP09lUm68vICwr56ZUNsjKZABBx8p5EKU6OrccJ3sQntpywYhvDZzAFh6A/ultUkBzlLvl6xOi93TI0oN/BDxCZHqesHNv2YS2h3JPTwFQLd5tRAh7O2iDDT1ZOb/JE2ZoPsnV+TAJIvyXDdz1SQ8UW5dlg6OZRyExBqdw5waBewHQD1bIV0E0bEq+7wAJPq049eQ0WTlnip04/Hf
```

Comment:
How many cases in each of these files?  If you haven't, consider a very small number of one case type to start. Then run bigger batches with files you definitely know need it.